### ISSUE TITLE
UPI tab not visible!!!! 

### ISSUE THREAD

**Issue by Rahat95** [razorpay/razorpay-android-sample-app#372]
as i said except UPI Payments all payment options are visible like cards, netbanking n all when i click on pay....... i have taken the same code from razor pay docs.
i m using springboot for serverside code

**Comment by padeepz** [comment]
same issue. expecially when trying to pay via mobile, UPI option is not showing, but when using desktop UPI is showing

**Comment by piyush-ramnani** [comment]
UPI mode would not be available unless you see 'Payment Methods' tab on your dashboard in 'Account and settings' option on the right. And that is available after submitting your KYC details. You don't have to wait for it to be verified, but need to complete it and 'Payment Methods' tab will be available. After that refresh your app and UPI option will show up.

After the KYC submission, you can test the UPI with the help of this:
https://razorpay.com/docs/payments/payments/test-upi-details/

Resolves #372

**Comment by vivekshindhe** [comment]
@padeepz can you share the ticket ID for the tickets you've raised on the dashboard? Will get this checked internally.

**Comment by padeepz** [comment]
@vivekshindhe #10569246 , #10626858, #10596485, #10095333, #9921385, #9863398.

**Comment by padeepz** [comment]
@vivekshindhe any progress ??

**Comment by padeepz** [comment]
@vivekshindhe pls any update, im not getting anything from support

**Comment by vivekshindhe** [comment]
@padeepz This should be resolved. I'm able to get the UPI applications for your MID. can you please check?

**Comment by padeepz** [comment]
@vivekshindhe yes, thanks a lot . UPI is working now in mobile. i really had this issue for 6+ months.
